docs/src/code/building-linuxcnc.adoc

```diff
@@ -5,34 +5,84 @@
 
 == Introduction
 
-This document describes how to build the LinuxCNC software and
-documentation from source.  This is primarily useful if you are a
-developer who is modifying LinuxCNC.  It can also be useful if you're
-a user who is testing developer branches, though then you also have
-the option of just installing Debian packages from the buildbot:
-http://buildbot.linuxcnc.org
+This document describes how to build the LinuxCNC software from source.
+This is primarily useful if you are a developer who is modifying LinuxCNC.
+It can also be useful if you're a user who is testing developer branches,
+though then you also have the option of just installing Debian packages from the buildbot
+(http://buildbot.linuxcnc.org) or as a regular package from your Linux
+distribution (https://tracker.debian.org/pkg/linuxcnc).
+Admittedly, this section also exists since LinuxCNC is a community effort.
+and you are encouraged to contribute to the development of LinuxCNC.
+You want to compile LinuxCNC yourself for The exchange of new developments,
```

Comment:
I suspect 'The exchange' should be spelled 'the exchange' with lower case t.  I find the phrase 'the exchange of new developments' a bit strange and an unknown idiom for me.  Perhaps 'Perhaps you want to compile LinuxCNC yourself to contribute to the development,' better express what you want?

Reply:
Addressed.